templates/index.html

```diff
@@ -44,8 +44,8 @@ <h2 class="panel-title">
                     <div id="collapseChangeCurriculum" class="panel-collapse collapse in">
                         <div class="panel-body">
                             <!--li>Change Module Quota</li>
-                            <li>Change Preclusions/Prerequisites</li>
-                            <li>Edit Module</li-->
+                            <li>Change Preclusions/Prerequisites</li-->
+                            <li><a id="home-page" href="/editAll">Edit All Mountings and Quotas</a></li>
```

Comment:
This addition here does not seem to be accounted for in `test_app.py`.

**Please add a test case for navigating to `/editAll`.**